apps/pie-docs/src/components/chip/overview/overview.md

```diff
@@ -0,0 +1,326 @@
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/variation-outline.svg",
+    alt: "The outline variant of the chip component.",
+    width: 63
+} %}
+
+### Ghost
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/variation-ghost.svg",
+    alt: "The ghost variant of the chip component.",
+    width: 63
+} %}
+
+---
+
+## Modifiers
+
+### Icon
+
+Icons are always positioned to the left of the string.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/modifier-icon.svg",
+    alt: "A chip component with a placeholder icon on the left.",
+    width: 87
+} %}
+
+### Tick icon
+
+Tick icon can be added for clarity and decision support when using chips to filter or multi-select to the selected state. 
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/modifier-tick-icon.svg",
+    alt: "A chip component with a tick icon on the left.",
+    width: 63
+} %}
+
+### Close icon
+
+Close icon can be added to the selected state to allow users to remove the selection or filter.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/modifier-close-icon.svg",
+    alt: "A chip component with a close icon on the right.",
+    width: 63
+} %}
+
+---
+
+## Size
+
+Chips have a minimum width of 48px.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/size.svg",
+    alt: "A chip component with the value of 1.",
+    width: 48
+} %}
+
+---
+
+## Content
+
+- Keep the strings short so they are easy to read and scan.
+- Use sentence case.
+
+---
+
+## Layout
+
+If there is a Chip grouping, by default are laid out horizontally and stack if required. A spacing of 8px is used horizontally, and 12px for vertical stacking.
+
+{% contentLayout %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/chip/layout-single-row.svg",
+      width: 237,
+      alt: "A group of chip components laid out horizontally",
+      variant: "secondary"
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/chip/layout-double-row.svg",
+      width: 237,
+      alt: "A group of chip components laid out horizontally and stacked",
+      variant: "secondary"
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Usage
+
+### Input
+
+Input chips represent information used in fields, such as an entity or different attributes.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/usage-input.svg",
+    alt: "Chip components used in an input field",
+    width: 256
+} %}
+
+### Choice
+
+In sets that contain at least two options, choice chips represent a single selection.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/usage-choice.svg",
+    alt: "Chip components representing various cuisines",
+    width: 297
+} %}
+
+### Filter
+
+In sets that contain at least two options, choice chips represent a single selection.
```

Comment:
It may be more of an application concern, but I think it would be good if we could spell out whether several chips can be selected at once, from a design perspective?

Reply:
@tingting-lau can you please advice here? ⭐

Reply:
"In sets that contain at least two options, each choice chip represents a single selection. Users can select more than one chip in the set."

P.S. this should be under "Choice" section. I've edited the text for "Filter" (left you a comment in figma). @raoufswe